resources/packs/core/tls.go

```diff
@@ -167,6 +168,11 @@ func (s *mqlTls) GetParams(socket Socket, domainName string) (map[string]interfa
 	if err != nil {
 		return nil, err
 	}
+	// listening on :: and passing that down as the host would fail, so
+	// make sure we format for ipv6
+	if strings.HasPrefix(host, "::") {
+		host = strings.Replace(host, "::", "[::]", 1)
```

Comment:
Just wondering whether we should change this already, where the address is created.

Looking at the output of `ss`, it also displays `[::]`:
```
$ sudo ss -tlnp
State           Recv-Q          Send-Q                   Local Address:Port                    Peer Address:Port          Process                                                                                   
LISTEN          0               511                               [::]:443                             [::]:*              users:(("nginx",pid=3849,fd=9),("nginx",pid=3848,fd=9),("nginx",pid=3847,fd=9))          
LISTEN          0               511                               [::]:80                              [::]:*              users:(("nginx",pid=3849,fd=7),("nginx",pid=3848,fd=7),("nginx",pid=3847,fd=7))
```

We could then show it here also as `[::]`:
```
ports.listening.where[1]: {
  tls: {
    domainName: ""
...
    socket: socket protocol="tcp" port=443 address="::"
```